### ISSUE TITLE
Scheme doesn't get built when trying Carthage install

### ISSUE THREAD

**Issue by richeterre** [JonasGessner/JGProgressHUD#36]
When trying to build the framework using Carthage v0.7.5, I see the following:

```
*** Checking out JGProgressHUD at "v1.2.7"
```

but nothing similar to this:

```
*** Building scheme "JGProgressHUD" in JGProgressHUD.xcworkspace
```

and afterwards there is no JGProgressHUD framework in Carthage's `Build` folder either. Can you reproduce this issue?

**Comment by lvillani** [comment]
I'm experiencing the same issue.

**Comment by lvillani** [comment]
If I remove `Carthage/Checkouts/JGProgressHUD/JGProgressHUD\ Tests/` I can get Carthage to build the framework. I guess that Carthage is trying to pick up the project from the tests directory.

**Comment by JonasGessner** [comment]
Ok so the problem is basically that there are two folders with the prefix "JGProgressHUD" which confuses Carthage, right? @lvillani if you rename the `Carthage/Checkouts/JGProgressHUD/JGProgressHUD\ Tests/` folder to something like `Carthage/Checkouts/JGProgressHUD/Tests/` does it work then?

**Comment by lvillani** [comment]
@JonasGessner That seems to work.

**Comment by JonasGessner** [comment]
Version 1.3 fixes this!

**Comment by lvillani** [comment]
I tried the version tagged as "v1.3" but it still doesn't build the framework for me. Not sure why.

**Comment by richeterre** [comment]
Yes, same here. I see the following console output:

```
*** Cloning Nimble
*** Cloning Quick
*** Cloning JGProgressHUD
*** Cloning SwiftyJSON
*** Cloning SnapKit
*** Cloning ReactiveCocoa
*** Cloning Result
*** Cloning Box
*** Downloading Quick at "v0.3.1"
*** Checking out ReactiveCocoa at "6c37b5ef416799a3a2a61ff78f051d5deb0d76b2"
*** Downloading Nimble at "Nimble v1.0.0 Final"
*** Downloading Box at "1.2.2"
*** Checking out JGProgressHUD at "v1.3"
*** Checking out SwiftyJSON at "2.2.0"
*** Checking out Result at "0.4.4"
*** Checking out SnapKit at "0.12.0"
*** xcodebuild output can be found in /var/folders/s3/ld_3x5y13w36s_bl0t11x56r0000gn/T/carthage-xcodebuild.BtDGFy.log
*** Building scheme "SnapKit iOS" in SnapKit.xcworkspace
*** Building scheme "SnapKit OSX" in SnapKit.xcworkspace
*** Building scheme "Result-iOS" in Result.xcworkspace
*** Building scheme "Result-Mac" in Result.xcworkspace
*** Building scheme "ReactiveCocoa iOS" in ReactiveCocoa.xcworkspace
*** Building scheme "ReactiveCocoa Mac" in ReactiveCocoa.xcworkspace
```
but the JGProgressHUD scheme isn't built, nor can I find the framework in Carthage's build folder.

**Comment by JonasGessner** [comment]
Weird.. I'll accept pull requests if you can fix it! I'm pretty busy at the moment.

**Comment by robertjpayne** [comment]
Pretty sure the issue is that there isn't an .xcodeproj file in the root directory? I don't think Carthage attempts to scan subdirectories…

**Comment by lvillani** [comment]
I'm not sure this is the case. In version 1.2.7 Carthage could find the project from within  the "JGProgressHUD" directory provided that you remove the "JGProgressHUD Tests" directory.

Carthage's source code is totally opaque to me to divine whichever rules it is using to search for Xcode projects.

**Comment by robertjpayne** [comment]
@lvillani I created a fork and moved the code to the root of the repo and it started working again, they may have decided it wasn't worth scanning subdirectories for project files?

**Comment by JonasGessner** [comment]
I have attached the prebuilt framework to the latest release (1.3). Carthage should use the prebuilt framework, I hope it works now!

**Comment by robertjpayne** [comment]
@JonasGessner that's not really a great solution, I actually turn off pre-built frameworks because some people are not properly distributing `dSYM` files with them and thus if a crash occurs inside the dependency you're out of luck to hunt it down.

**Comment by JonasGessner** [comment]
Ok. Damn, I guess I'll just move the xodeproj to the root directory.

**Comment by JonasGessner** [comment]
Quick update, also for #37 and #38: I've moved the xcodeproject to the root directory, I'll push an update next week or so. Thanks for the bug reports!

**Comment by alpennec** [comment]
Hi Jonas, any update on this please?

**Comment by JonasGessner** [comment]
I'll push the update as soon as I can get to it. I have terrible internet at the moment and I'm quite busy but I should be able to push the update next week!

**Comment by JonasGessner** [comment]
Fixed in 01dfbc2c3400d80e165f66d03e816b72f9ef3fff.
